app/subscriber/src/features/my-searches/hooks/useElastic.ts

```diff
@@ -10,18 +11,19 @@ import { generateMustNotQuery, generateQuery, IFilterSettingsModel, Settings } f
  */
 export const useElastic = () => {
   const [{ isReady, settings }] = useLookup();
-
-  const [frontPageImagesMediaTypeId, setFrontPageImagesMediaTypeId] = React.useState(0);
+  const [{ frontPageImagesMediaTypeId }, { storeSettingsFrontPageImagesMediaTypeId }] =
```

Comment:
Moved logic to redux to reduce the visibility of how horrible React state is.  Now only the very first call will have the issue.  Before it would be every time this hook was used the first time.